api/task.go

```diff
@@ -38,16 +42,19 @@ func newTaskHandler(store *event.Store, drivers *driver.Drivers,
 
 // ServeHTTP serves the tasks endpoint
 func (h *taskHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	log.Printf("[TRACE] (api.task) requesting tasks '%s'", r.URL.Path)
+	logger := logging.FromContext(r.Context())
+	logger.Trace("requesting tasks", "url_path", r.URL.Path)
 
 	switch r.Method {
 	case http.MethodPatch:
 		h.updateTask(w, r)
 	default:
 		err := fmt.Errorf("'%s' in an unsupported method. The task API "+
 			"currently supports the method(s): '%s'", r.Method, http.MethodPatch)
-		log.Printf("[TRACE] (api.task) unsupported method: %s", err)
-		jsonErrorResponse(w, http.StatusMethodNotAllowed, err)
+		logger.Trace("unsupported method", "error", err)
+		if err = jsonErrorResponse(w, http.StatusMethodNotAllowed, err); err != nil {
+			logger.Error("error, could not generate json error response", "error", err)
```

Comment:
Since this is already logged on the error level + the key value "error", I think it would be better to remove the `error,` at the beginning of the log message

Reply:
and similar with the other occurrences of this log